release-notes/opensearch-ml-common.release-notes-2.14.0.0.md

```diff
@@ -9,6 +9,7 @@ Compatible with OpenSearch 2.14.0.0
 
 ### Enhancements
 * Change httpclient to async ([#1958](https://github.com/opensearch-project/ml-commons/pull/1958))
+* Fix user defined preprocess function missing prediction issue ([#2418](https://github.com/opensearch-project/ml-commons/pull/2418))
```

Comment:
is that a enhancement or bug fix?

Reply:
It's a bug fix but it's a `new feature bug fix` instead of `released feature bug fix`, putting it in bug fix part is confusing to customers.

Reply:
We don't have any categorization between `new feature bug fix` vs `released feature bug fix`, so why that'll be confusing? A bug fix is always a bug fix regardless of the feature status IMO.

Reply:
I see, makes sense. Thanks for the explanation!

Reply:
So this is a bug fix for a new feature being shipped for the first time? Just skip the changelog entry so it's not confusing.

Reply:
@dblock that's right, do you mean we don't need to add this in the release note(close the PR)?

Reply:
Close this PR as 2.14 already released.